src/index.html

```diff
@@ -11,8 +12,74 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <!-- #endregion -->
+
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <h1 class="imac">
+      <img
+        src="images/Rectangle 4.png"
+        alt="mac"
+      />
+    </h1>
+
+    <h2 class="h2">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+    <div class="product-code">Product code: 195434</div>
+
+    <div class="reviews">
+      <div class="stars">
+        <img
+          src="images/star.svg"
+          alt="star"
+        />
+        <img
+          src="images/star.svg"
+          alt="star"
+        />
+        <img
+          src="images/star.svg"
+          alt="star"
+        />
+        <img
+          src="images/star.svg"
+          alt="star"
+        />
+        <img
+          src="images/star-active.svg"
+          alt="star"
+        />
+      </div>
+      <div class="reviews__revs">Reviews: 5</div>
+    </div>
+
+    <div class="price">
+      <div class="price__text">Price:</div>
+      <div class="price__amount">$2,199</div>
+    </div>
+
+    <div
```

Comment:
The class name 'button' is too generic for a `div` element wrapping a button. Consider using 'button-wrapper' or a similar descriptive name.